tests/integration/conftest.py

```diff
@@ -206,9 +207,13 @@ async def nginx_integrator_app_fixture(
             "nginx-ingress-integrator",
             application_name=nginx_integrator_app_name,
             trust=True,
-            channel="latest/edge",
+            channel="latest/stable",
+            revision=121,
+        )
+        # The nginx-ingress-integrator charm goes into "waiting" when waiting for relation
+        await model.wait_for_idle(
```

Comment:
Isnt that because is waiting for the ingress IP?

Reply:
Not quite, the status message explicitly says that it's waiting for the relation. I had to change this because tests were timing out as they were expecting the charm to be in an active state.

Reply:
Ah just found an example
https://github.com/canonical/nginx-ingress-integrator-operator/actions/runs/11501212954/job/32014000618#step:21:45